features/transactions-grouping.feature

```diff
@@ -8,8 +8,7 @@ Feature: Transactions Grouping
   Background:
     Given I have selected English language
     And I have accepted "Terms of use"
-    And I agree to send logs to remote server
-    And I have a wallet
+    And I have a wallet with funds
```

Comment:
I have no idea how this step worked before ;)

Reply:
@darko-mijic it did not work at all - it is skipped (notice the `@skip` line on the top of the file) 😱